src/main/java/fi/otavanopisto/kuntaapi/server/integrations/gtfs/GtfsIdFactory.java

```diff
@@ -0,0 +1,38 @@
+package fi.otavanopisto.kuntaapi.server.integrations.gtfs;
+
+import fi.otavanopisto.kuntaapi.server.id.AbstractIdFactory;
+import fi.otavanopisto.kuntaapi.server.id.OrganizationId;
+import fi.otavanopisto.kuntaapi.server.id.PublicTransportAgencyId;
+import fi.otavanopisto.kuntaapi.server.id.PublicTransportRouteId;
+import fi.otavanopisto.kuntaapi.server.id.PublicTransportScheduleId;
+import fi.otavanopisto.kuntaapi.server.id.PublicTransportStopId;
+import fi.otavanopisto.kuntaapi.server.id.PublicTransportStopTimeId;
+import fi.otavanopisto.kuntaapi.server.id.PublicTransportTripId;
+
+public class GtfsIdFactory extends AbstractIdFactory {
+  
+  public PublicTransportAgencyId createAgencyId(OrganizationId kuntaApiOrganizationId, String gtfsAgencyId) {
+    return createId(PublicTransportAgencyId.class, kuntaApiOrganizationId, GtfsConsts.IDENTIFIER_NAME, String.format("%s-%s", kuntaApiOrganizationId.getId(), gtfsAgencyId));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "%s-%s" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)